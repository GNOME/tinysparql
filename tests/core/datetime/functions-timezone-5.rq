SELECT TIMEZONE (?v)
WHERE {
	example:x a example:A ;
	   example:p ?v
}
