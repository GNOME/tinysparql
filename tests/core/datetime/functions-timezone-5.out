"-PT12H34M"
